### ISSUE TITLE
Add Monthly View to Expends Statistics Page

### ISSUE THREAD

**Issue by sonalder-darlene** [sweakpl/unlock-master#11]
Currently, the app allows users to view daily unlock statistics, but only for a 7-day streak. While this feature is great, I would love to see my progress over a longer period. Specifically, it would be beneficial to have a monthly view that allows me to track my evolution week by week, and potentially even a yearly view to analyze trends month by month.

Your app is fantastic as it is functional and has a great user interface. I purchased the pro version and have no regrets (if anything, I feel I should have paid more for it, like $3). However, the insights are currently limited to day-to-day analysis, and I prefer a broader perspective that includes week-to-week and month-to-month comparisons without the need to scroll or export data.

I believe adding these features would greatly enhance the app's usability and provide users with a more comprehensive understanding of their unlocking habits. Thank you for considering this suggestion!

**Comment by sweakpl** [comment]
Hey, thank you for suggestion! I will give it a look definitely, although I cannot guarantee when the feature might be expanded,

**Comment by tian2992** [comment]
Yes, month, 3m, and 6m would be amazing, specially to see trends and habits in more detail!
